src/includes/performance/tracingOrigins-example/javascript.nextjs.mdx

```diff
@@ -0,0 +1,18 @@
+For example:
+
+- A frontend application is served from `example.com`
+- A backend service is served from `api.example.com`
+- The frontend application makes API calls to the backend
+- Set the `tracingOrigins` option to `['api.example.com']`
+- Now outgoing XHR/fetch requests to `api.example.com` will get the `sentry-trace` header attached
```

Comment:
```suggestion
- A frontend application is served from `example.com`.
- A backend service is served from `api.example.com`.
- The frontend application makes API calls to the backend.
- Set the `tracingOrigins` option to `['api.example.com']`.
- Now outgoing XHR/fetch requests to `api.example.com` will get the `sentry-trace` header attached.
```
?

Reply:
Yeah, fair. I don't love sentences as bullet points in the first place, but you're right, they are full sentences for the moment.